### DISCUSSION TITLE
LEVEL_RANGE

### DISCUSSION THREAD

**Discussion by BinbinZhou-NOAA** [dtcenter/METplus#1557] [Configuration] [answered]
In PB2NC's  MET config file,  there are elevation_range and level_range settings.
What is their difference?  While in PB2NC's METPlus conf, only PB2NC_LEVEL_RANGE_BEG 
and PB2NC_LEVEL_RANGE_END) are defined,  but no ELEVATION_RANGE is defined?

**Comment by CPKalb** [comment, marked as answer]
Hi Binbin:

The elevation refers to the elevation of a station observation, whereas level would be the level in the atmosphere (such as 500mb or 250mb).  Right now, ELEVATION_RANGE is not available in METplus but can be specified directly in a configuration file, whereas the level range is available in METplus.  Do you need ELEVATION_RANGE to be available in METplus?

Thanks,
Christina

**Reply by BinbinZhou-NOAA** [reply]
Christina,

  Thanks for the explanation. Not require for ELEVATION_RANGE right now.

Binbin

**Reply by CPKalb** [reply]
Okay, great.  I'm going to mark this as answered then.  If you have more questions, feel free to start a new discussion
